### ISSUE TITLE
!SC and !SD are missing on the T-Lite / !SH is also missing from the X9D+ and TX16S

### ISSUE THREAD

**Issue by raphaelcoeffic** [EdgeTX/edgetx#178]
Mentioned in #175

**Comment by raphaelcoeffic** [comment]
@pfeerick I think this is on purpose for 2 positions switches, as it is considered redundant. This is true, but a bit confusing at the same time, as one wonders why the switch are not in that list.

bottom line: we can change this very easily, but it is probably a matter of taste. I leave this to you.

**Comment by raphaelcoeffic** [comment]
Here is for reference the relevant code:
```
    frame #0: 0x00000001000e5aa6 simu`isSwitchAvailable(swtch=24, context=MixesContext) at gui_common.cpp:322:11
   319 	      return false;
   320 	    }
   321 	    if (!IS_CONFIG_3POS(swinfo.quot)) {
-> 322 	      if (negative) {
   323 	        return false;
   324 	      }
   325 	      if (swinfo.rem == 1) {
```

**Comment by pfeerick** [comment]
Ah... ok... that makes sense... the ! states are important for the 3POS switches, but are redundant... I've give that some thought then, and if there's valid reasons to add them as well, I'll do a PR. I'll leave this issue open, and target it to be resolved one way or the other in 2.4.1 - as I think it either needs to be documented, or allowed.

**Comment by pfeerick** [comment]
Verdict of the poll was 

![image](https://user-images.githubusercontent.com/5500713/130710064-06757190-6e31-486c-82f9-6aef6fd07234.png)

so it looks like this is going ahead...
